src/ape/api/transactions.py

```diff
@@ -318,22 +317,17 @@ def await_confirmations(self) -> "ReceiptAPI":
     def method_called(self) -> Optional[MethodABI]:
         """
         The method ABI of the method called to produce this receipt.
-        Requires both that the user is using a provider that supports traces
-        as well as for this to have been a contract invocation.
         """
 
-        call_tree = self.call_tree
-        if not call_tree:
-            return None
-
-        return self.chain_manager.contracts.lookup_method(call_tree.address, call_tree.calldata)
```

Comment:
Ya I removed it on this PR, it is no longer needed.

We could deprecate it but I think since it is a breaking change and probably not a common method, it is probably ok to delete for 0.6.

Reply:
Sounds good to me